cartographer_ros/cartographer_ros/start_trajectory_main.cc

```diff
@@ -59,11 +59,14 @@ bool Run() {
           kStartTrajectoryServiceName);
   cartographer_ros_msgs::StartTrajectory srv;
   srv.request.options = ToRosMessage(LoadOptions());
-  srv.request.topics.laser_scan_topic = kLaserScanTopic;
-  srv.request.topics.multi_echo_laser_scan_topic = kMultiEchoLaserScanTopic;
-  srv.request.topics.point_cloud2_topic = kPointCloud2Topic;
-  srv.request.topics.imu_topic = kImuTopic;
-  srv.request.topics.odometry_topic = kOdometryTopic;
+  srv.request.topics.laser_scan_topic =
```

Comment:
Please add `true /* explain boolean meaning */.

Reply:
Just for the first one, right?